internal/acceptance/lakeview_dashboard_test.go

```diff
@@ -0,0 +1,412 @@
+package acceptance
+
+import (
```

Comment:
In general, you should check that the resource has been correctly created in the workspace. You can use the Go SDK client to access the dashboard.